index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <link href="css/style.css" rel="stylesheet" type="text/css" />
+        <title>Apple</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    </head>
+    <body>
+        <header class="header row">
+            <nav>
+                <ul>
+                    <li class="logo"><a href="#" ><img src="img/logo.png" alt="apple-logo"></a></li>
+                    <li><a href="#">Store</a></li>
+                    <li><a href="#">Mac</a></li>
+                    <li><a href="#">iPod</a></li>
+                    <li><a href="#">iPhone</a></li>
+                    <li><a href="#">iPad</a></li>
+                    <li><a href="#">iTunes</a></li>
+                    <li><a href="#">Support</a></li>
+                    <li><input type="search" /></li>
+                </ul>
+            </nav>
+            <div></div>
+            <h1>
+                <a href="#">What will your verse be?</a>
+            </h1>
+            <h2>
+                <a  href="#">Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.
+                </a>
+            </h2>
+            <h5>
+                <a  href="#">Watch the “Your Verse” film and explore the stories</a>
+            </h5>
+        </header>
+        <div class="row">
+            <div class="col-3 mobile-imag1"></div>
+            <div class="col-3 mobile-imag2"></div>
+            <div class="col-3 mobile-imag3"></div>
+            <div class="col-3 mobile-imag4"></div>
```

Comment:
please wrap each one of these images inside a link tag, like the original page.

Reply:
done